__tests__/PropTypesDevelopmentReact15.js

```diff
@@ -825,6 +825,20 @@ describe('PropTypesDevelopmentReact15', () => {
       typeCheckPass(PropTypes.oneOf('red', 'blue'), 'red');
     });
 
+    it('should warn but not error for invalid multiple arguments', () => {
```

Comment:
Why is it better to warn here, and not throw an error? Wouldn’t that surface it most rapidly?

Reply:
I totally agree - it's my first contribution to `prop-types` so I'm being cautious here. I didn't want the output of neither `oneOf('red')` nor `oneOf('red', 'blue')` to change, as this may be considered breaking by some. So I just "enhanced" an error message in one of the cases.